app/main.py

```diff
@@ -124,6 +124,7 @@ def run_one_task(task: Task) -> bool:
     except Exception as e:
         log.log_exception(logger, e)
         run_status_message = f"Task {task_id} failed with exception: {e}."
+        logger.handlers.clear()
```

Comment:
What's the reason for adding this change?

Reply:
See: https://github.com/nus-apr/auto-code-rover/issues/19